release-tools/SIDECAR_RELEASE_PROCESS.md

```diff
@@ -102,7 +102,7 @@ naming convention `<hostpath-deployment-version>-on-<kubernetes-version>`.
 1. If release was a new major/minor version, create a new `release-<minor>`
    branch at that commit.
 1. Check [image build status](https://k8s-testgrid.appspot.com/sig-storage-image-build).
-1. Promote images from k8s-staging-sig-storage to k8s.gcr.io/sig-storage. From
+1. Promote images from k8s-staging-sig-storage to registry.k8s.io/sig-storage. From
```

Comment:
Changes in release-tools should be pulled by using subtree in a different PR.  It should not be part of this PR.

Reply:
get~